### ISSUE TITLE
Impossible names

### ISSUE THREAD

**Issue by donno2048** [freeminer/freeminer#286]
I can't use `space` or nothing as a name when creating a new world, it would be great if when you don't type any name it will create an auto-name, something like "NewWorld0" or something

**Comment by seungju398** [comment]
Yes, that would be very helpful and simple way to start a new world for every player.
